galera/src/replicator_smm.hpp

```diff
@@ -295,12 +295,12 @@ namespace galera
 #ifdef PXC
             int is_eof() { return eof_; }
 #endif /* PXC */
-            void reset() { eof_ = false; error_ = 0; processed_upto_ = 0;}
-            void eof(int error)
+            void reset() { eof_ = false; result_ = ist::Result{0, ""}; }
```

Comment:
`processed_upto_ = 0` should be added as well
```suggestion
            void reset() { eof_ = false; result_ = ist::Result{0, ""}; processed_upto_ = 0; }
```

Reply:
Done